### ISSUE TITLE
The preset is loaded but not applied

### ISSUE THREAD

**Issue by Flone-dnb** [wwmm/easyeffects#1046]
I'm running Manjaro with KDE and can't make legacy PulseEffects autostart properly. The service is started and the preset is loaded. In my preset, I only have an equalizer, but it's not applied. You have to open a GUI instance of PulseEffects tick the checkbox on the equalizer to turn it off and then turn it on again to make it work. Closing the GUI instance after this does not reset the processing or something - all works.

P. S. Easyeffects not working - no processing is happening no matter what I do.

Any tips? :)

**Comment by Flone-dnb** [comment]
`pulseeffects --gapplication-service && pulseeffects -r && pulseeffects -l *preset_name*` does the same.

**Comment by vchernin** [comment]
EasyEffects (the succesor to PulseEffects) only supports PipeWire. You need to have PipeWire install as your audio backend to use it, [see here for arch](https://wiki.archlinux.org/title/PipeWire#Installation).

PulseEffects is in maintenance mode, so it's unlikely there will be a lot of work on it I think.

**Comment by Flone-dnb** [comment]
I've had bad luck switching to PipeWire on this device, this is why I'm using legacy PulseEffects.

**Comment by Flone-dnb** [comment]
More to this issue: I've noticed that `pulseeffects -q` before shutdown fixes the issue, but I was not able to automate this as it was not helping for some reason.

**Comment by wwmm** [comment]
Kill the current instance `pulseeffects -q` and restart it in debug mode `G_MESSAGES_DEBUG=pulseeffects pulseeffects`. This should show what is happening. But I am confused. The preset does not load only when using `pulseeffects -l preset_name` or it does not work even when using the presets menu in our window?

**Comment by Flone-dnb** [comment]
Well, first, I use the "Load" button from the app window to load the preset. Then, I tick the "Start Service at Login" to autostart and reboot. When rebooted, my preset (that only has an equalizer) is not applied (I can hear the difference), but the service is running. So, I first thought that I also need to `pulseeffects -l preset_name` in autostart, and so I added that but it has no effect.

After a reboot, I open PulseEffects window and see that my preset is loaded with all my settings and the equalized is enabled (the checkbox is checked). But only after I tick off and tick on the checkbox on the equalizer, only after that, I can hear that the equalizer is actually applied. I have really heavy settings on the equalizer so the difference is very obvious.

`G_MESSAGES_DEBUG=pulseeffects pulseeffects` output:
```
(process:2339): pulseeffects-DEBUG: 14:45:02.712: main: locale directory: /usr/share/locale
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.756: presets_manager: system input presets directory: "/etc/PulseEffects/input"; 
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.756: presets_manager: system input presets directory: "/etc/xdg/PulseEffects/input"; 
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.756: presets_manager: system input presets directory: "/home/flone/.config/kdedefaults/PulseEffects/input"; 
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.756: presets_manager: system output presets directory: "/etc/PulseEffects/output"; 
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.756: presets_manager: system output presets directory: "/etc/xdg/PulseEffects/output"; 
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.756: presets_manager: system output presets directory: "/home/flone/.config/kdedefaults/PulseEffects/output"; 
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.756: presets_manager: user presets directory already exists: /home/flone/.config/PulseEffects
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.756: presets_manager: user presets directory already exists: /home/flone/.config/PulseEffects/input
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.756: presets_manager: user presets directory already exists: /home/flone/.config/PulseEffects/output
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.756: presets_manager: user presets directory already exists: /home/flone/.config/PulseEffects/autoload
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.759: application: PE version: 4.8.4
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.759: pulse_manager: context is connecting
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.760: pulse_manager: context is authorizing
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.760: pulse_manager: context is setting name
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.760: pulse_manager: context is ready
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.760: pulse_manager: connected to: /run/user/1000/pulse/native
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.760: pulse_manager: protocol version: 34
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.760: pulse_manager: Pulseaudio version: 14.2
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.760: pulse_manager: default pulseaudio source: alsa_input.pci-0000_00_1f.3.analog-stereo
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.760: pulse_manager: default pulseaudio sink: alsa_output.pci-0000_00_1f.3.analog-stereo
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.760: pulse_manager: loading Pulseeffects applications output sink...
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.760: pulse_manager: default pulseaudio sink sampling rate: 48000 Hz
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.760: pulse_manager: default pulseaudio sink audio format: s16le
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.761: pulse_manager:  failed to get sink info: PulseEffects_apps
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.765: pulse_manager: loaded module-null-sink: sink_name=PulseEffects_apps sink_properties=device.description="PulseEffects(apps)"device.class="sound" norewinds=1
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.765: pulse_manager: changed volume of the sink: PulseEffects_apps
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.766: pulse_manager: loading Pulseeffects microphone output sink...
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.766: pulse_manager: default pulseaudio source sampling rate: 44100 Hz
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.766: pulse_manager: default pulseaudio source audio format: s16le
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.766: pulse_manager:  failed to get sink info: PulseEffects_mic
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.769: pulse_manager: loaded module-null-sink: sink_name=PulseEffects_mic sink_properties=device.description="PulseEffects(mic)"device.class="sound" norewinds=1
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.769: pulse_manager: changed volume of the sink: PulseEffects_mic
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.787: sie: using input device: PulseEffects_apps.monitor
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.787: sie: using output device: alsa_output.pci-0000_00_1f.3.analog-stereo

(pulseeffects:2339): pulseeffects-WARNING **: 14:45:02.787: limiter plugin was not found!

(pulseeffects:2339): pulseeffects-WARNING **: 14:45:02.793: filter plugin was not found!

(pulseeffects:2339): pulseeffects-WARNING **: 14:45:02.837: reverb plugin was not found!

(pulseeffects:2339): pulseeffects-WARNING **: 14:45:02.837: bass_enhancer plugin was not found!

(pulseeffects:2339): pulseeffects-WARNING **: 14:45:02.837: exciter plugin was not found!

(pulseeffects:2339): pulseeffects-WARNING **: 14:45:02.839: maximizer plugin was not found!

(pulseeffects:2339): pulseeffects-WARNING **: 14:45:02.839: multiband_compressor plugin was not found!

(pulseeffects:2339): pulseeffects-WARNING **: 14:45:02.841: gate plugin was not found!

(pulseeffects:2339): pulseeffects-WARNING **: 14:45:02.841: pitch plugin was not found!

(pulseeffects:2339): pulseeffects-WARNING **: 14:45:02.841: multiband_gate plugin was not found!

(pulseeffects:2339): pulseeffects-WARNING **: 14:45:02.842: deesser plugin was not found!

(pulseeffects:2339): pulseeffects-WARNING **: 14:45:02.842: stereo_tools plugin was not found!

(pulseeffects:2339): pulseeffects-WARNING **: 14:45:02.850: rnnoise plugin was not found!
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.855: soe: using output device: PulseEffects_mic
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.855: soe: using input device: alsa_input.pci-0000_00_1f.3.analog-stereo

(pulseeffects:2339): pulseeffects-WARNING **: 14:45:02.855: limiter plugin was not found!

(pulseeffects:2339): pulseeffects-WARNING **: 14:45:02.856: filter plugin was not found!

(pulseeffects:2339): pulseeffects-WARNING **: 14:45:02.861: reverb plugin was not found!

(pulseeffects:2339): pulseeffects-WARNING **: 14:45:02.861: gate plugin was not found!

(pulseeffects:2339): pulseeffects-WARNING **: 14:45:02.861: deesser plugin was not found!

(pulseeffects:2339): pulseeffects-WARNING **: 14:45:02.862: pitch plugin was not found!

(pulseeffects:2339): pulseeffects-WARNING **: 14:45:02.866: multiband_compressor plugin was not found!

(pulseeffects:2339): pulseeffects-WARNING **: 14:45:02.866: multiband_gate plugin was not found!

(pulseeffects:2339): pulseeffects-WARNING **: 14:45:02.866: stereo_tools plugin was not found!

(pulseeffects:2339): pulseeffects-WARNING **: 14:45:02.866: maximizer plugin was not found!

(pulseeffects:2339): pulseeffects-WARNING **: 14:45:02.866: rnnoise plugin was not found!
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.868: presets_manager: system input presets directory: "/etc/PulseEffects/input"; 
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.868: presets_manager: system input presets directory: "/etc/xdg/PulseEffects/input"; 
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.868: presets_manager: system input presets directory: "/home/flone/.config/kdedefaults/PulseEffects/input"; 
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.868: presets_manager: system output presets directory: "/etc/PulseEffects/output"; 
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.868: presets_manager: system output presets directory: "/etc/xdg/PulseEffects/output"; 
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.868: presets_manager: system output presets directory: "/home/flone/.config/kdedefaults/PulseEffects/output"; 
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.868: presets_manager: user presets directory already exists: /home/flone/.config/PulseEffects
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.868: presets_manager: user presets directory already exists: /home/flone/.config/PulseEffects/input
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.868: presets_manager: user presets directory already exists: /home/flone/.config/PulseEffects/output
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.868: presets_manager: user presets directory already exists: /home/flone/.config/PulseEffects/autoload
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.868: presets_manager: destroyed
pulseeffects-INFO: 14:45:02.868: application: disabling global bypass
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.868: sie: bypass is already disabled
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:02.868: soe: bypass is already disabled
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:03.283: convolver_ui: irs directory already exists: /home/flone/.config/PulseEffects/irs
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:03.297: rnnoise_ui: model directory already exists: /home/flone/.config/PulseEffects/rnnoise
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:03.728: rnnoise_ui: model directory already exists: /home/flone/.config/PulseEffects/rnnoise
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:03.772: sie: spectrum enabled
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:03.772: soe: spectrum enabled
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:04.383: sie: sampling rate: 48000 Hz
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:04.399: sie: using output device: alsa_output.pci-0000_00_1f.3.analog-stereo
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:04.399: pulse_settings_ui: output device changed: alsa_output.pci-0000_00_1f.3.analog-stereo
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:04.399: pulse_settings_ui: added sink: alsa_output.pci-0000_00_1f.3.analog-stereo
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:04.399: pulse_settings_ui: added source: alsa_output.pci-0000_00_1f.3.analog-stereo.monitor
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:04.400: soe: using input device: alsa_input.pci-0000_00_1f.3.analog-stereo
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:04.400: pulse_settings_ui: input device changed: alsa_input.pci-0000_00_1f.3.analog-stereo
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:04.400: pulse_settings_ui: added source: alsa_input.pci-0000_00_1f.3.analog-stereo
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:04.400: sie: NULL -> READY -> PLAYING
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:04.401: sie: READY -> PAUSED -> PLAYING
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:04.402: sie: pulsesrc latency [us]: 0
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:04.402: sie: pulsesrc buffer [us]: 200000
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:04.402: sie: total latency: 0 ms
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:04.402: sie: pulsesink latency [us]: 0
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:04.402: sie: pulsesink buffer [us]: 200000
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:04.402: sie: total latency: 0 ms
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:04.813: sie: total latency: 140 ms
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:04.818: sie: PAUSED -> PLAYING -> VOID_PENDING
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:04.818: sie: total latency: 140 ms
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:07.941: presets_manager: autoloading preset pulse_effects_hd_280_config for device alsa_output.pci-0000_00_1f.3.analog-stereo:analog-output-speaker
(pulseeffects:2339): pulseeffects-DEBUG: 14:45:07.950: presets_manager: loaded preset: /home/flone/.config/PulseEffects/output/pulse_effects_hd_280_config.json
```
The preset is loaded but I don't hear it being applied, I then disable/enable the EQ:
```
(pulseeffects:3084): pulseeffects-DEBUG: 14:49:59.283: sie: equalizer is already disabled
(pulseeffects:3084): pulseeffects-DEBUG: 14:49:59.842: sie: equalizer is enabled
```
The `equalizer is already disabled` part is weird because the UI shows that it was enabled actually.
Here is the EQ part from the config:
`"equalizer": {
            "state": "true",`
I've now tried creating a fresh preset with EQ - same result.
Also noticed that I can't change params of most of the effects (like reverb) even when they are enabled.

**Comment by wwmm** [comment]
>PE version: 4.8.4

Upgrade to `4.8.5`. When the distributions moved to `dconf >= 0.40` I had to update the legacy package. I think that what you are seeing now is the same bug that was fixed in 4.8.5

**Comment by Flone-dnb** [comment]
Switched to the AUR version `4.8.5-3` and it fixed the issue! The version that I had: `community/pulseeffects-legacy 4.8.4-4`. I probably need to tell the people about this.

Thanks a lot!
